packages/xsnap/test/test-xs-perf.js

```diff
@@ -1,4 +1,4 @@
-/* global performance */
+/* global performance, issueCommand */
```

Comment:
`issueCommand` is not a global. It's only available in the XS start Compartment. So keeping the scope small is worthwhile.

Reply:
Do you mean it won't be available for the entirety of this module execution? If so, I agree. If not, defining at the file scope is more accurate, no?

Reply:
Correct; it won't be available for the entirety of this module execution. The function in which it _looks_ like it is used is actually stringified and sent to XS for execution. This merits an explicit comment, clearly.